src/EoS/Nuclear/CPU_EoS_Nuclear.cpp

```diff
@@ -217,8 +238,15 @@ static real EoS_DensEint2Pres_Nuclear( const real Dens_Code, const real Eint_Cod
    real Temp_MeV  = NULL_REAL;
    real Entr      = NULL_REAL;
    real Pres_CGS  = NULL_REAL;
+   real mu_nu     = NULL_REAL;
    real Useless   = NULL_REAL;
+
    int  Err       = NULL_INT;
+#ifdef FLOAT8
+   const real Tolerance = 1e-10;
+#else
+   const real Tolerance = 1e-6;
+#endif
 
```

Comment:
Would it be better to define `Tolerance` in `NuclearEoS.h`?

Reply:
I agree with you. I'd better to do that.

Reply:
I corrected the typo and removed unnecessary space in `NuclearEoS.h`